src/Agent/NewRelic/Agent/Core/BrowserMonitoring/BrowserScriptInjectionHelper.cs

```diff
@@ -25,7 +25,7 @@ public static async Task InjectBrowserScriptAsync(byte[] buffer, Stream baseStre
             {
                 // not found, can't inject anything
                 transaction?.LogFinest("Skipping RUM Injection: No suitable location found to inject script.");
-                await baseStream.WriteAsync(buffer, 0, buffer.Length);
+                await baseStream.WriteAsync(buffer, 0, buffer.Length).ConfigureAwait(false);
```

Comment:
Are we sure that this is safe to do? It seems like we would need some additional testing to ensure that this doesn't cause a problem.